mpc/mpc-aio/src/protocol/ot/kos/sender.rs

```diff
@@ -120,6 +127,87 @@ impl ObliviousSend<[Block; 2]> for Kos15IOSender<s_state::RandSetup> {
     }
 }
 
+// The idea is to send AES encryption keys in the OT, which can then later be used by the receiver
```

Comment:
// The idea for obliviously sending a vec of Blocks is to send an AES encryption key using OT, which
// can then later be used by the receiver to decrypt an arbitrary long message, which is sent shortly
// after the OT. This way we extend our OT from a 128-bit maximum message length to an unlimited message
// length.